server/controllers/buyer/addUserOrdersById.js

```diff
@@ -0,0 +1,29 @@
+const { throwError } = require('../../utilities');
+const { addOrdersById } = require('../../database/queries');
+
+const addUserOrdersById = (req, res, next) => {
+  const {
+    productId,
+    quantity,
+    price,
+    paymentMethod,
+  } = req.body;
+  const { userId } = req.params;
+
+  if (quantity < 1) {
+    throw throwError(400, 'quantity must be greater than 1');
+  } else {
+    const orderNumber = `${Date.now()}${productId}‏${userId}`;
+    addOrdersById(
+      productId,
+      quantity,
+      price,
+      paymentMethod,
+      orderNumber,
+      userId,
```

Comment:
please make validation on the received data before inserting it in the database,
for example when no quantity is sent it will be saved as null in your database

![image](https://user-images.githubusercontent.com/32321796/107141547-077f5a00-6932-11eb-951b-9d2965306093.png)

![image](https://user-images.githubusercontent.com/32321796/107141540-fa626b00-6931-11eb-8ae6-e9a345322edd.png)